src/coffee/directives/api/utils/fit-helper.coffee

```diff
@@ -9,5 +9,6 @@ angular.module('uiGmapgoogle-maps.directives.api.utils')
           everSet = true unless everSet
           point = if _.isFunction markerOrPoint.getPosition then markerOrPoint.getPosition() else markerOrPoint
         bounds.extend point
-      gMap.fitBounds(bounds) if everSet
+      if everSet
+        $timeout () -> gMap.fitBounds(bounds)
```

Comment:
Why is `$timeout` really required. Understanding `$timeout` does a setTiemout and a digest / apply. So in reality your just forcing a digest to happen later. This could be done with `$evalAsync`, but then you would need a scope object.